test/unit_tests/compressorFncsTest.cpp

```diff
@@ -47,17 +47,28 @@ TEST_F(CompressorFncsTest, compressorSpecs)
     {
         // get preset model
         HPWH hpwh;
-        EXPECT_EQ(hpwh.initPreset(modelSpec.sModelName), 0)
+        EXPECT_NO_THROW(hpwh.initPreset(modelSpec.sModelName))
```

Comment:
We can probably just remove the check here (and in most places where we're using `EXPECT_NO_THROW` where we were checking for zero return codes. If the method throws, the test will fail.